tezos_interop/tezos_interop.re

```diff
@@ -348,6 +348,7 @@ module Pack = {
   type t = node(canonical_location, prim);
 
   let int = n => Int(-1, n);
+  let nat = n => Int(-1, n);
   let bytes = b => Bytes(-1, b);
   let pair = (l, r) => Prim(-1, D_Pair, [l, r], []);
   let list = l => Seq(-1, l);
```

Comment:
For these, the first parameter (`-1`) is discarded during packing, so it doesn't affect the hash, which is all we care about, right?

Reply:
yup, it's the original location on the micheline, I think it's used only for error messages